lua/graph.lua

```diff
@@ -1474,6 +1475,8 @@ function filter_tags_generic(kv)
         use = 27
      elseif kv["highway"] == "pedestrian" then
         use = 28
+     elseif kv["highway"] == "platform" then
+        use = 35
```

Comment:
this branch needs to be before the next one. in fact, I wonder if the `bridleway` is not too low in the hierarchy? if a bridleway only has `pedestrian_forward` at this stage it doesn't get a bridleway use.